tests/common.go

```diff
@@ -7581,3 +7581,182 @@ func GetClusterProviders() []string {
 	}
 	return clusterProviders
 }
+
+func IsPoolOffline(n node.Node) (bool, error) {
+	poolsStatus, err := Inst().V.GetNodePoolsStatus(n)
+	if err != nil {
+		return false, err
+	}
+
+	for _, v := range poolsStatus {
+		if v == "Offline" {
+			return true, nil
+		}
+	}
+	return false, nil
+}
+
+func GetPoolUuidsWithStorageFull() ([]string, error) {
```

Comment:
delete this if not required

Reply:
keeping this function for now , will delete or modify the test later